### PR TITLE
Feat/add GitHub repo stars info

### PR DESCRIPTION
**Pull request by mr-fox93**

### CODE REVIEW

File: src/components/GithubLoginButton/GithubLoginButton.module.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.iconBox {
-  width: 14px;
-  height: 14px;
-}
+// .iconBox {
```
Comment (Sumick) [inline src/components/GithubLoginButton/GithubLoginButton.module.scss:1]: **nit:** Let's remove this file if not needed

File: src/components/landing-page/LandingHeader/LandingHeader.tsx [hunk 2]
```diff
@@ -37,9 +49,19 @@ const LandingHeader = async () => {
...
+              <GitHubButtonClient />
+            </div>
+          </div>
           <div className={styles.frameButtons}>
             <div className={styles.buttonBoxDesktop}>
+              <div>
+                {' '}
+                <GitHubButtonClient />
+              </div>{' '}
+              {/* <div>Button</div>{' '} */}
```
Comment (Sumick) [inline src/components/landing-page/LandingHeader/LandingHeader.tsx:64]: comment to remove

**Review by Sumick — APPROVED** [review]
Good job, 2 cleanup related comments

**Review by wawrzonnn — APPROVED** [review]

**Review by szymonkadas — APPROVED** [review]
Gj

**Review by AniaPiWo — APPROVED** [review]

**Pull request merged**